press/api/client.py

```diff
@@ -331,6 +331,9 @@ def is_owned_by_team(doctype, docname, raise_exception=True):
 	if not frappe.local.team():
 		return False
 
+	if frappe.local.system_user():
+		return True
```

Comment:
this makes the function name return a "wrong" value, why is this needed?

Reply:
Removed it and added impersonate button instead.